vnext/Chakra/ChakraJsiRuntime_core.cpp

```diff
@@ -5,22 +5,21 @@
 
 #include "ChakraJsiRuntime.h"
 #include "unicode.h"
+#include "ChakraJsiRuntimeFactory.h"
 
 #include <cxxreact/MessageQueueThread.h>
 
 // This file contains non-edge-mode (or win32) implementations.
 #if !defined(USE_EDGEMODE_JSRT)
 #include <ChakraCore.h>
 
-#if !defined(JSI_CORE)
 #define CHAKRART_HAS_DEBUGGER
```

Comment:
Is this define and the include of ChakraCoreDebugger.h intended to be for ChakraCore and not Chakra (for UWP)?

Reply:
This is just removing the JSI_CORE define, which is no longer used. Functionality remains intact otherwise.

---
In reply to: [283583657](https://github.com/microsoft/react-native-windows/pull/2461#discussion_r283583657) [](ancestors = 283583657)